features/admin.applications.v1/components/forms/general-details-form/general-details-form.tsx

```diff
@@ -274,6 +274,8 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
         error: groupsListFetchError
     } = useGetGroupsMetadata(selectedUserStoreDomain, searchTerm);
 
+    const isFragmentApp: boolean = application?.advancedConfigurations?.fragment || false;
```

Comment:
Shall we rename the variable as follows?

```suggestion
    const isSharedApp: boolean = application?.advancedConfigurations?.fragment || false;
```

Reply:
Done